public/app/plugins/panel/calendar/plugin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "panel",
+  "name": "Calendar",
+  "id": "calendar",
+  "state": "beta",
+  "visualisationType": "calendar",
```

Comment:
This will be required for panel to be shown in explore, otherwise there would not be a way to match the panel to the data.

Reply:
Do we need to name this to something more specific to explore? in case this might be confused with something not-explore related?

Reply:
I mentioned before that I don't think this should be explore prefixed because the concept even though used in explore right now isn't specifically tied to explore. If we decide to use this value somewhere else like for visualisation suggestions in dashboards having an Explore prefix will be confusing and as this is a public API changing it at that moment may be breaking change. So this represents what is the type of visualisation and and that should reflect the naming how it is used may evolve over time so the name should reflect just the primary idea behind it not a current downstream usage imho.